modApiServer/test/org/aion/api/server/ApiAionTest.java

```diff
@@ -321,7 +321,6 @@ public void testEstimates() {
         ArgTxCall txcall = new ArgTxCall(addr, Address.ZERO_ADDRESS(),
                 msg, repo.getNonce(addr), BigInteger.ONE, 100000, 100000);
 
-        assertNotEquals(0, api.estimateGas(txcall));
```

Comment:
are you sure this test should be removed? Isn't `estimateGas` supposed to do the same thing as what `estimateNrg` now does?